frontend/packages/console-app/src/components/quick-starts/utils/useQuickStarts.ts

```diff
@@ -0,0 +1,33 @@
+import { useAccessReview } from '@console/internal/components/utils';
+import { useFlag } from '@console/shared/src/hooks/flag';
+import { QuickStart } from './quick-start-types';
+import { getQuickStarts } from './quick-start-utils';
+
+const useQuickStarts = (): QuickStart[] => {
+  const quickStarts = getQuickStarts();
+
+  const filteredQuickStarts = quickStarts.filter((quickStart) => {
+    const {
+      spec: { accessReviewResources, flags },
+    } = quickStart;
+
+    const hasAccess =
+      !accessReviewResources ||
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      accessReviewResources.map((descriptor) => useAccessReview(descriptor)).every((x) => x);
```

Comment:
`useAccessReview` uses `useAccessReview2` which use `useSafetyFirst` which use `useState` internally. Disabling this eslint rule here works only because the quickStarts are statically saved and the order does never change until we change this.

Can you please add an comment here why you disable the eslint rules in this case? WDYT? Then its fine for me.

Reply:
Done.